src/rsz/test/repair_fanout8.ok

```diff
@@ -185,6 +185,172 @@ Iteration | Removed | Resized | Inserted | Cloned Gates | Pin Swaps |   WNS   |
       229 |       0 |      24 |       45 |            0 |         1 |  -5.170 | -389.486 | load117/D
       230 |       0 |      24 |       45 |            0 |         1 |  -5.170 | -389.486 | load117/D
       230 |       0 |      24 |       45 |            0 |         1 |  -5.170 | -389.486 | load117/D
+      230 |       0 |      24 |       45 |            0 |         1 |  -5.170 | -389.486 | load117/D
```

Comment:
All the new iterations past here don't accomplish anything - an opportunity to tune the termination criteria in the future.

Reply:
Yes.  Agree.